src/GitUtilities.js

```diff
@@ -38,7 +38,8 @@ export default class GitUtilities {
 
   static addFile(file, opts) {
     log.silly("addFile", file);
-    ChildProcessUtilities.execSync("git", ["add", file], opts);
+    const portablePath = path.relative(opts.cwd, path.resolve(opts.cwd, file)).replace(/\\/g, '/');
```

Comment:
It's a pity there's no built-in way to coerce paths to posix... (`path.posix.*` methods don't work with Windows paths, near as I can tell)

Reply:
@evocateur 

Would it be worthwhile to pull in one of these dependencies?

https://www.npmjs.com/package/path-posix
https://www.npmjs.com/package/ensure-posix-path (no README but tests look good)